modules/core/04-channel/types/msgs_test.go

```diff
@@ -350,14 +351,14 @@ func (suite *TypesTestSuite) TestMsgChannelCloseConfirmValidateBasic() {
 		msg     *types.MsgChannelCloseConfirm
 		expPass bool
 	}{
```

Comment:
super super nit, but should we add a success test case for non zero counterparty upgrade sequence?